### ISSUE TITLE
Zxing handleResult not called in Nexus 5X-7.1.1

### ISSUE THREAD

**Issue by rramprasad** [dm77/barcodescanner#280]
I am using barcode scanner latest version, 

> compile 'me.dm7.barcodescanner:zxing:1.9'

It works great on other devices except Nexus 5X(7.1.1) device.I doesn't called handleResult on the Fragment. I debug inside the ZXingScannerView.class and always I got `com.google.zxing.NotFoundException` on following line,

```
try {
       rawResult = mMultiFormatReader.decodeWithState(bitmap);
     } catch (ReaderException re) {
         // continue
         Log.d("ZxingScannerView","ReaderException ->"+re.toString());
}
```

**Comment by annanoell** [comment]
Hi @rramprasad  , I'm facing the same issue. Could you fix it?
I'm using the ZXingScannerView inside a fragment, and the ResultHandler is never called when the ScannerView is inside a layout (the scanner might be half screen for my app). But it works if you place the code in the middle of the screen even if you cannot see the code.

Have you found a solution for this?

Thank you.

**Comment by rramprasad** [comment]
@annanoell  Refer my #284

**Comment by rramprasad** [comment]
Fixed this issue,Refer https://github.com/dm77/barcodescanner/pull/315

**Comment by rekhaagarwal** [comment]
@rramprasad @dm77 has the above issue fixed in latest version? facing the same issue on Nexus 5x on latest release 1.9.3. Can you please tell me when the fix will be released.

Thanks in advance. :)

**Comment by dm77** [comment]
Fixed in 1.9.8 finally. Thanks @rramprasad for the pull request
